digitaltwins/samples/device/typescript/src/sample_device.ts

```diff
@@ -13,7 +13,6 @@
 import { DigitalTwinClient, CommandCallback, CommandRequest, CommandResponse, PropertyChangedCallback, BaseInterface } from 'azure-iot-digitaltwins-device';
 import { EnvironmentalSensor } from './environmentalinterface';
 import { DeviceInformation } from './deviceInformationInterface';
-import { SampleExit } from './exitInterface';
 
```

Comment:
This is unfortunate.  I rather liked this example.

Reply:
All interfaces should be modeled with DTDL and them implemented for each language. We can discuss to add it in the future. What did you like about it?

Reply:
It was short, actionable, easy to understand, useful

---
In reply to: [413970433](https://github.com/Azure/azure-iot-sdk-node/pull/768#discussion_r413970433) [](ancestors = 413970433)

Reply:
But it looks like you don't want it so I'll close this.

---
In reply to: [414044301](https://github.com/Azure/azure-iot-sdk-node/pull/768#discussion_r414044301) [](ancestors = 414044301,413970433)